source/common/upstream/outlier_detection_impl.cc

```diff
@@ -208,12 +234,89 @@ void DetectorImpl::onConsecutive5xxWorker(HostSharedPtr host) {
   ejectHost(host, EjectionType::Consecutive5xx);
 }
 
+const double Utility::SUCCESS_RATE_STDEV_FACTOR = 1.9;
+
+double Utility::successRateEjectionThreshold(double success_rate_sum,
+                                             std::vector<double>& success_rate_data) {
```

Comment:
You did comment on this already. I missed it writing the big comment after the function signature.